cmd/create/cluster/cmd.go

```diff
@@ -201,7 +201,8 @@ func init() {
 		"Prefix to use for all IAM roles used by the operators needed in the OpenShift installer. "+
 			"Leave empty to use an auto-generated one.",
 	)
-
+	flags.MarkDeprecated("operator-roles-prefix", "use rosa create operator-roles -c cluster-id "+
+		"instead")
```

Comment:
Change the text to `"use 'rosa create operator-roles' command instead"`, since we don't have the cluster ID. Users often copy/paste these outputs :/